server/ladder_service.py

```diff
@@ -111,3 +232,12 @@ def inform_player(self, player: Player):
 
             # Collect all the rows from the ResultProxy
             return [row[game_stats.c.mapId] async for row in await conn.execute(query)]
+
+    def on_connection_lost(self, player):
+        self.cancel_search(player)
```

Comment:
Good point. Since this PR doesn't have parties yet it isn't an issue, but we will need to think about how this cancelling logic works. Obviously if any player in a party disconnects we need to make sure it cancels the entire party search.